spec/rubocop/cli/cli_autocorrect_spec.rb

```diff
@@ -1653,6 +1653,30 @@ def self.some_method(foo, bar: 1)
     RUBY
   end
 
+  it 'corrects Style/BlockDelimiters offenses when specifing' \
+     'Layout/SpaceBeforeBlockBraces with `EnforcedStyle: no_space` together' do
+    create_file('example.rb', <<~RUBY)
+      foo {
+        bar
+      }
+    RUBY
+
+    create_file('.rubocop.yml', <<~YAML)
+      Layout/SpaceBeforeBlockBraces:
+        EnforcedStyle: no_space
+    YAML
+
+    expect(cli.run(%w[--auto-correct])).to eq(0)
```

Comment:
Please don't let this example depend on other cops than `--only Style/BlockDelimiters,Layout/SpaceBeforeBlockBraces`. This is to prevent unnecessary work in the future if any of the unrelated cops changes. I remember 8d071f0c9a32f594b696cb7a3b6a0e5adabad42b all to well …

Reply:
Thanks for the review. I updated it.